src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Lifted/nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
```

Comment:
Do we need this nuget.config or is the default fine?

Reply:
yes, this should not be needed